SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -1222,7 +1209,7 @@ if ($PsBoundParameters.ContainsKey("startup")) {
 
         # Startup the NSX Manager Nodes in the Management Workload Domain
         Start-CloudComponent -server $vcServer.fqdn -user $vcUser -pass $vcPass -nodes $nsxtNodes -timeout 600
-        if (!(Wait-ForStableNsxtClusterStatus -server $nsxtMgrfqdn -user $nsxMgrVIP.adminUser -pass $nsxMgrVIP.adminPassword)) {
+        if (!(Wait-ForStableNsxtClusterStatus -server $nsxtManagerFQDN -user $nsxtManagerVIP.adminUser -pass $nsxtManagerVIP.adminPassword)) {
```

Comment:
I've seen the use of `if (!(...` and `if (-Not`.

Might be a good idea to ensure it's consistent with either.